src/cplscheme/BaseCouplingScheme.hpp

```diff
@@ -403,11 +397,27 @@ class BaseCouplingScheme : public CouplingScheme {
    */
   bool accelerate();
```

Comment:
This name is confusing. We should rename this to `iterate()` or `doImplicitStep()`, since currently we either `performAcceleration` or `extrapolateData` here. Acceleration is only a part of what is happening in this function.

Reply:
Done in a694b88